### ISSUE TITLE
Remake plot commands to enable arithmetic expressions

### ISSUE THREAD

**Issue by markusrobertjonsson** [learningsimulator/learningsimulator#103]
**1. Add the syntax `@plot v(s->v)` as an alternative to**
```
    @vplot s->v
```
Analogously,
```
    @plot w(s1)
    @plot v(s1->b1)
    @plot p(s1->b1)
    @plot vss(s1->s2)
    @plot n(s1->b1->s2->...)
```
and also for export
```
    @export w(s1)
    @export v(s1->b1)
    @export p(s1->b1)
    @export vss(s1->s2)
    @export n(s1->b1->s2->...)
```

This makes it possible to make arithmetic operations, for example
```
    @plot v(s1->b1) + v(s1->b2)
    @plot p(s1->b1) * p(s1->b2)
    @plot n(s->response) / n(s)
```

**2. Also allow wildcard syntax in plot commands:**
```
@plot p(red -> *)
@plot p(* -> *)
@plot w(*)
...
```

Also allow multiple plots in the same plot command:
```
@plot p(red->r) ; p(red->x)  # Makes both @pplot(red->r) and @pplot(red->x)
@pplot red[0.1],blue[0.3]->r ; red->x  # Makes both @pplot(red[0.1],blue[0.3]->r) and @pplot(red->x)
```

With any asterisk syntax, we do not allow setting label, in other words, we always get default label whenever any asterisk is used.

**3. Remake @nplot with cumulative off (to be cumulative value within each interval given by xscale). To make estimation of p, use**
```
@plot n(red->response) / n(red)
```

**Comment by markusrobertjonsson** [comment]
We could even allow
```
@plot p(red -> *) * p(blue -> *)
```
but with any asterisk syntax, we do not allow setting label, in other words, we always get default label whenever any asterisk is used.

**Comment by markusrobertjonsson** [comment]
Remake @nplot with cumulative off (to be cumulative value within each intrval given by xscale). To make estimation of p, use
```
@nplot(red->response) / n(response)
```

**Comment by drghirlanda** [comment]
To make multiple plots, I think this is a bit weird:
`@plot p(red->r, red->x)`
because it looks like there is a p function that takes two arguments. Perhaps this is more clear:
`@plot p(red->r), p(red->x)`

**Comment by drghirlanda** [comment]
I think this syntax in 3 is hard to understand:
`@nplot(red->response) / n(red)`
because it looks like the whole @nplot command is divided by n(red). I think this would be clearer:
`@nplot n(red->response) / n(red)`
or also
`@plot n(red->response) / n(red)`

**Comment by markusrobertjonsson** [comment]
> To make multiple plots, I think this is a bit weird: `@plot p(red->r, red->x)` because it looks like there is a p function that takes two arguments. Perhaps this is more clear: `@plot p(red->r), p(red->x)`
Yes, and also since we allow compund stimulus to pplot: `@pplot e1[0.5],e2[0.5]->r`. There could be a comma-conflict. I will change the description.

**Comment by markusrobertjonsson** [comment]
This was fixed in release 1.1.
